modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -483,47 +305,20 @@ public function add_appearance_menu( $domain, $calypso = true ) {
 
 	/**
 	 * Adds Plugins menu.
-	 *
-	 * @param string $domain  Site domain.
 	 */
-	public function add_plugins_menu( $domain ) {
+	public function add_plugins_menu() {
 		remove_submenu_page( 'plugins.php', 'plugin-editor.php' );
```

Comment:
This one seems counterintuitive, `add_plugins_menu()` just to `remove_submenu_page( 'plugins.php', 'plugin-editor.php' );`. Is there any other better place to do this?